### PR TITLE
Context menu additions / show source on click

### PR DESCRIPTION
**Pull request by Samirat**
I really wanted to be able to run/debug from the test explorer context menu, and also show source when I clicked on tests. The second one is an amenity, and the first one lets you run arbitrary subsets of tests, which I couldn't find a way to do before. Let me know what you think.

The context menu additions are a good addition, I think, the open source on click thing some people might not want. Could stick it behind a setting or something.

Great extension by the way.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "icon": "icon.png",
   "author": "Holger Benl <hbenl@evandor.de>",
   "publisher": "hbenl",
-  "version": "2.17.0",
+  "version": "99.0.0",
```
Comment (Gama11) [inline package.json]: Huh? :D

Comment (Samirat) [reply package.json]: haha whoops

Comment (Samirat) [reply package.json]: Reverted

**Comment by Trass3r** [comment]
Does this also make "Run all tests" from the context menu jump to the right node in the test tree (option "Show on run" is enabled)?

**Comment by Samirat** [comment]
> Does this also make "Run all tests" from the context menu jump to the right node in the test tree (option "Show on run" is enabled)?

Don't think so, this right click options to run tests, and adds an option so when you left click on a test in the explorer, it will jump to the test in the editor.

**Comment by Samirat** [comment]
@hbenl Can you take a look at this?

**Pull request closed without merging**